sklearn/metrics/tests/test_regression.py

```diff
@@ -23,6 +24,7 @@ def test_regression_metrics(n_samples=50):
     y_pred = y_true + 1
 
     assert_almost_equal(mean_squared_error(y_true, y_pred), 1.)
+    assert_almost_equal(mean_squared_log_error(y_true, y_pred), 0.01915163)
```

Comment:
@jnothman Done ! I was skeptical about the fact that if `mean_squared_error` actually gets faulty, then these tests will still pass as we are doing the same thing internally.